### PR TITLE
Standardizing Asset Directory Structure

### PR DESCRIPTION
**Pull request by zakhaev26**
### Description
This PR addresses the issue related to the inconsistent storage of image and video assets in the Talawa API repository. The goal was to standardize the directory structure to align with the one used in Talawa Admin.

### Changes Made
1. Moved image and videos assets from the root directory to `src/assets/image` and `src/assets/video` following the structure used in Talawa Admin.
2. Updated markdown files in the root directory to reference the new asset locations under `src/assets`.
3. Deleted assets files from the root directory that were not referenced in markdown files at `talawa-api` .
- Currently the entire codebase didn't have any reference of assets present in `videos` directory ,so I removed it.
Screenshot from 2023-12-03 10-00-09

5. Configured `.gitignore` to prevent future uploads to `video/` and `image/` directories and ensure these directories remain in the repository.
6. Checked and updated objects in markdown files to reference the new directory structure.
- Current Structure : 
image

### Links
- [Talawa Admin Asset Structure](https://github.com/PalisadoesFoundation/talawa-admin/tree/main/src/assets)
- [Original Issue](https://github.com/PalisadoesFoundation/talawa-api/issues/1441)

### Additional Notes

- This PR is part of the effort to maintain a consistent and organized asset directory in the codebase across Talawa repositories.

### Potential Impact
Developers working on the Talawa API should be aware of the changes in asset locations.

**Closes: #1441**

### CODE REVIEW

**Review by palisadoes — CHANGES_REQUESTED** [review]
We need an answer for this question here before we can approve this PR. The correct file locations need to be verified. We **_MUST NOT_** put files in an inappropriate location.

- https://github.com/PalisadoesFoundation/talawa-admin/pull/1112#issuecomment-1839388641

File: .gitignore [hunk 1]
```diff
@@ -33,3 +33,6 @@ csr.pem
 
 .DS_STORE
 
+# Ensuring Images are not uploaded in ./image , ./video path 
+.image
```
Comment (palisadoes) [inline .gitignore:37]: This is incorrect. The directory needs to be maintained but no files can be uploaded using git.

**Review by palisadoes — CHANGES_REQUESTED** [review]
The Gitignore is incorrect.

File: .gitignore (hunk 1 above)
Comment (zakhaev26) [reply .gitignore:37]: > The directory needs to be maintained but no files can be uploaded using git.
So should I also keep the copy of assets currently being used in the repo in the maintained directory as well?

**Comment by palisadoes** [comment]
Not if they are unused elsewhere

**Comment by zakhaev26** [comment]
PTAL

**Comment by zakhaev26** [comment]
> instead of mentioning each file in gitignore we can have separate folder for the images we need in the project.

won't that allow commits to images folder ?
As I was asked that the directory (images,videos in root ) needs to be maintained , but no files can be uploaded in that folder using git.

**Comment by noman2002** [comment]
> Changes looks good to me, but instead of mentioning each file in gitignore we can have separate folder for the images we need in the project.

@palisadoes Your thoughts on this.

**Comment by zakhaev26** [comment]
> If the image/ and videos/ directories are required by the code base, then they should remain, else they should be removed.

The `image/` and `videos/` directory is not required in the codebase now ,as they have been shifted to the `public` directory.
Redundant files have been already deleted and all the references to these assets ahve been changed and are valid too.
In that case,is it necessary to keep the these `image/` and `videos/` folder?

File: .gitignore [hunk 2]
```diff
@@ -33,3 +33,7 @@ csr.pem
 
 .DS_STORE
 
+# Ensuring no images and video are commited in ./video and ./image directories
+
+
+markdown/*
```
Comment (EshaanAgg) [inline .gitignore:39]: Why have you added the markdown directory here?
It should contain the `videos` and `images` directories that get the uploaded files.

**Review by EshaanAgg — CHANGES_REQUESTED** [review]

**Comment by palisadoes** [comment]
1. Please merge with the latest upstream and commit. That should remove the errors.
2. We need to get this implemented. The outcome will be used in 2024 for some issues in the pipeline

**Comment by palisadoes** [comment]
Is this ready for review?

**Comment by palisadoes** [comment]
Closing due to inactivity

**Pull request closed without merging**